cmake/sip_configure.py

```diff
@@ -100,7 +100,7 @@ def __init__(self):
 
 
 def custom_platform_lib_function(self, clib, framework=0):
-    if os.path.isabs(clib):
+    if os.path.isabs(clib) or clib.startswith('-l'):
```

Comment:
Please add a comment describing why this check is necessary and libraries that already append -l